R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -832,10 +846,10 @@ test_that("test HiveContext", {
   expect_equal(count(df2), 3)
 
   jsonPath2 <- tempfile(pattern="sparkr-test", fileext=".tmp")
-  saveAsTable(df, "json", "json", "append", path = jsonPath2)
-  df3 <- sql(hiveCtx, "select * from json")
+  invisible(saveAsTable(df, "json2", "json", "append", path = jsonPath2))
```

Comment:
this is to avoid printing NULL in the console. Maybe not necessary within `testthat`. But I think it does not hurt here.